Adapters/Vungle/ISVungleAdapter/ISVungleRewardedVideoAdapterRouter.m

```diff
@@ -0,0 +1,125 @@
+//
+//  ISVungleRewardedVideoAdapterRouter.m
+//  ISVungleAdapter
+//
+//  Copyright © 2020 ironSource. All rights reserved.
+//
+
+#import "ISVungleRewardedVideoAdapterRouter.h"
+#import "ISVungleAdapter.h"
+
+@interface ISVungleRewardedVideoAdapterRouter()
+
+@property (nonatomic, strong, nullable) NSString *bidPayload;
+
+@end
+
+@implementation ISVungleRewardedVideoAdapterRouter
+
+- (instancetype)initWithPlacementID:(NSString *)placementID
+                      parentAdapter:(ISVungleAdapter *)parentAdapter
+                           delegate:(id<ISRewardedVideoAdapterDelegate>)delegate {
+    if (self = [super init]) {
+        _placementID = placementID;
+        _parentAdapter = parentAdapter;
```

Comment:
u are not using this parentAdapter anywhere.
My suggestion is create a Close metod in the parent adapter class and call that method  inside rewardedAdDidClose.
u should do this in all the ad formates. Parent adapter should have its own close call for each adFormate and in side that close call u should nil the Router instanse.

Reply:
@ManojBudumuru-vungle I removed the parentAdapter property in each ad type AdapterRouter.  Please refer to below comment for more details. Thanks a lot. :)
https://github.com/Vungle/levelplay-ios-adapters/pull/1#discussion_r992330578